cms/static/cms/js/modules/cms.modal.js

```diff
@@ -547,29 +698,50 @@
             },
 
             /**
-             * sanitise the ampersand within the url for #3404
+             * Sanitise the ampersand within the url for #3404.
              *
+             * @method _prepareUrl
              * @param url {String}
+             * @private
              */
-            _prepareUrl: function (url) {
+            _prepareUrl: function _prepareUrl(url) {
                 // FIXME: A better fix is needed for '&' being interpreted as the
                 url = url.replace('&', '&amp;');
                 return url;
             },
 
-            _loadContent: function (url, name) {
+            /**
+             * Version where the modal loads an iframe.
+             *
+             * @method _loadIframe
+             * @param opts
+             * @param opts.url {String} url to render iframe, takes presedence over opts.html
+             * @param [opts.breadcrumbs] {Object[]} collection of breadcrumb items
+             * @param [opts.title] {String} modal window main title (bold)
+             */
+            _loadIframe: function _loadIframe(opts) {
                 var that = this;
-                url = this._prepareUrl(url);
+
+                opts.url = this._prepareUrl(opts.url);
+                opts.title = opts.title || '';
+                opts.breadcrumbs = opts.breadcrumbs || '';
+
+                // set classes
+                this.ui.modal.removeClass('cms-modal-content');
```

Comment:
can argue that `markup` is better than `content` in this case, because content is a bit too generic, but both okay for me